src/tree/updater_quantile_hist.cc

```diff
@@ -902,8 +1249,14 @@ void QuantileHistMaker::Builder::EnumerateSplit(int d_step,
         }
```

Comment:
```cpp
best.Update(loss_chg, fid, split_pt, true, c, e);
```
since `d_step` is known to be -1 in this scope.

Reply:
Done